.gitmodules

```
@@ -2,3 +2,6 @@
 	path = lib/aave-helpers
 	url = https://github.com/bgd-labs/aave-helpers
 	branch = main
+[submodule "lib/aave-umbrella"]
+	path = lib/aave-umbrella
+	url = https://github.com/bgd-labs/aave-umbrella
```

```suggestion
	url = https://github.com/aave-dao/aave-umbrella
```
actually better to add the DAO repo no?

---

No, I deployed *slightly* different versions with interfaces 0.8.27 versions, on our repo I downgraded them to 0.8.0, cause proposals don't compile for versions more than 0.8.22 with some internal foundry error